### PR TITLE
SWW-2368 - Added functionality to recieve a nice filename from API an…

### PR DESCRIPTION
**Pull request by richmoh**
…d use that for the file names

### CODE REVIEW

File: storage/file_test.go [hunk 1]
```diff
@@ -37,7 +37,28 @@ func TestFile_GetPathInZip(t *testing.T) {
...
+		assert.Equal(t, test.want, file.GetRelativePath())
+	}
+}
+
+func TestFile_GetFileNameAndExtension(t *testing.T) {
+	tests := []struct {
+	    scenario        string
+		filename        string
+		wantFilename    string
+		wantExtension   string
```
Comment (richmoh) [inline storage/file_test.go:49]: Indentation gone a bit wierd here. Mix of tabs and spaces?

File: storage/file_test.go [hunk 2]
```diff
@@ -37,7 +37,28 @@ func TestFile_GetPathInZip(t *testing.T) {
...
+	}{
+		{"Filename with one dot", "filename.txt", "filename", "txt"},
+        {"Filename with mutiple dots", "filename.something.txt", "filename.something", "txt"},
+        {"Filename with no file extension", "filename", "filename", ""},
```
Comment (richmoh) [inline storage/file_test.go:53]: And here

File: storage/file_test.go (hunk 1 above)
Comment (devkokov) [reply storage/file_test.go:49]: yeah, I'm thinking of using this tool to fix indentations throughout the codebase, but probably best done in a separate PR, do you agree? - https://blog.golang.org/gofmt

**Comment by richmoh** [comment]
@devkokov Feel free to approve this

**Pull request merged**